server/src/main/java/org/elasticsearch/index/mapper/ParametrizedFieldMapper.java

```diff
@@ -229,6 +231,16 @@ public boolean isConfigured() {
             return this;
         }
 
+        /**
+         * Set a serialization predicate for this parameter.
+         *
+         * The predicate is passed the value of `include_defaults`
+         */
+        public Parameter<T> setSerializerPredicate(BooleanSupplier serializerPredicate) {
```

Comment:
Maybe change the name of this method? `shouldSerializeDefault` or something?

Reply:
++, I've changed it to `setShouldSerialize` and made the javadoc a bit clearer.